src/components/Person/Person.jsx

```diff
@@ -1 +1,17 @@
-// export const Person = ({ person }) => ();
+export const Person = ({
+  person: { name, age = null, sex, isMarried, partnerName },
+}) => (
+  <section className="Person">
+    <h2 className="Person__name">My name is {name}</h2>
+    {age !== null && <p className="Person__age">I am {age}</p>}
+    <p className="Person__partner">
+      {isMarried === true && sex === 'm' ? <p>{partnerName} is my wife</p> : ''}
+      {isMarried === true && sex === 'f' ? (
+        <p>{partnerName} is my husband</p>
+      ) : (
+        ''
+      )}
```

Comment:
```suggestion
      {isMarried && sex === 'f' 
        ? (
          <p>{partnerName} is my husband</p>
        ) 
        : (
          ''
        )}
```

Formatting, variables, indents